packages/vanilla/src/sass/components/tabs/_index.scss

```diff
@@ -0,0 +1,58 @@
+.cbp-tabset {
+  display: flex;
+  align-items: flex-end;
+  //gap: var(--cbp-space-2x);
+  border-bottom: var(--cbp-border-size-md) solid var(--cbp-color-gray-cool-30);
+  margin-bottom: var(--cbp-space-4x);
+
+  .cbp-tab {
+    all: unset;
+    min-height: var(--cbp-space-14x);
+    padding: 0 var(--cbp-space-3x);
+    margin-bottom: calc(var(--cbp-space-half-x) * -1);
+    border-bottom: var(--cbp-border-size-lg) solid transparent;
```

Comment:
Mockups say the border-bottom of a hovered/active tab should be 4px width

Reply:
Fixed. I thought I had the right token, but must have miscalculated. Nice catch.